go/client/cmd_device_remove.go

```diff
@@ -49,14 +53,35 @@ func (c *CmdDeviceRemove) Run() (err error) {
 
 	return cli.RevokeDevice(context.TODO(), keybase1.RevokeDeviceArg{
 		Force:    c.force,
-		DeviceID: c.id,
+		DeviceID: id,
 	})
 }
 
+func (c *CmdDeviceRemove) lookup(name string) (keybase1.DeviceID, error) {
+	cli, err := GetDeviceClient()
+	if err != nil {
+		return "", err
+	}
+	if err := RegisterProtocols(nil); err != nil {
+		return "", err
+	}
```

Comment:
I think the cleanest thing would be to just do a single `RegisterProtocols` call at the top of `Run`, and not bother doing it again here. @patrickxb do you know off the top of your head why most of our commands seem to call `Get***Client()` *before* they call `RegisterProtocols(...)`? Does the order matter for anything there?